gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -992,8 +992,8 @@
      * Adds a {@link Vertex} with a default vertex label.
      *
      * @return the traversal with the {@link AddVertexStep} added
-     * @since 3.1.0-incubating
      * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#addvertex-step" target="_blank">Reference Documentation - AddVertex Step</a>
+     * @since 3.1.0-incubating
```

Comment:
did your IDE do all this javadoc annotation switching? or was that intentional? not something to switch back or anything i guess, but it just made it hard to root out what's changed here of substance and what hasn't.